src/pages/landing/stocks/_parallelogram.js

```diff
@@ -0,0 +1,169 @@
+import React from 'react'
+import styled from 'styled-components'
+import PropTypes from 'prop-types'
+import { graphql, useStaticQuery } from 'gatsby'
+import device from 'themes/device'
+import { Container, SectionContainer } from 'components/containers'
+import { Header, Text, QueryImage } from 'components/elements'
+
+const StyledSection = styled(SectionContainer)`
+    background-color: var(--color-grey-30);
+    clip-path: polygon(0 0, 100% 6%, 100% 100%, 0 94%);
+    padding: 6.4rem 1.6rem;
+    margin: 2rem auto;
+`
+const StyledContainer = styled(Container)`
+    display: flex;
+    flex-direction: column;
+`
+const Content = styled.div`
```

Comment:
```suggestion
const Content = styled(Flex)`
```

We can use the `Flex` component instead

Reply:
Changed div to flex